docs/config.md

```diff
@@ -578,6 +578,21 @@ as follows:
 signing.backends.ssh.allowed-signers = "/path/to/allowed-signers"
 ```
 
+## Commit Signature Verification
+
+Commit verification uses the configured [signing backends](#commit-signing) to attempt to verify
+any signatures found on commits when displaying them with `jj log` or `jj show`.
```

Comment:
"configured signing backends" here sounds like "enabled signing backends" - which is not the case, the verification uses "all the backends" - and well while you have to configure allowed signers for ssh, for gpg it actually just works™ if it's installed and the keys are imported (and even for unknown keys it'll show key fingerprint), no jj configuration needed